# HG changeset patch
# User Wormnest
# Date 1376484409 -7200
#      Wed Aug 14 14:46:49 2013 +0200
# Node ID 61839fdd7a62ec238334ba7003f4e2bf4dce0af2
# Parent  c8f3c4fc0e83c0ffd15edbb3b48b7c9309fb70fb
Add: Introduce strings needed for filtering the news history.

diff -r c8f3c4fc0e83 -r 61839fdd7a62 src/lang/english.txt
--- a/src/lang/english.txt	Tue Aug 13 17:45:19 2013 +0000
+++ b/src/lang/english.txt	Wed Aug 14 14:46:49 2013 +0200
@@ -745,6 +745,27 @@
 # News message history
 STR_MESSAGE_HISTORY                                             :{WHITE}Message History
 STR_MESSAGE_HISTORY_TOOLTIP                                     :{BLACK}A list of the recent news messages
+STR_MESSAGE_HISTORY_ALL_VEHICLES                                :{BLACK}All types of vehicles
+STR_MESSAGE_HISTORY_ALL_CARGOS                                  :{BLACK}All cargos
+
+# News history types
+STR_NEWS_ALL_TYPES                                              :All news
+STR_NEWS_ARRIVAL_COMPANY                                        :First vehicle for company
+STR_NEWS_ARRIVAL_OTHER                                          :First vehicle for competitor(s)
+STR_NEWS_ACCIDENT                                               :Accidents or disasters
+STR_NEWS_COMPANY_INFO                                           :Company info
+STR_NEWS_INDUSTRY_OPEN                                          :Opening of industries
+STR_NEWS_INDUSTRY_CLOSE                                         :Closing of industries
+STR_NEWS_ECONOMY                                                :Economic changes
+STR_NEWS_INDUSTRY_COMPANY                                       :Production changes of company
+STR_NEWS_NDUSTRY_OTHER                                          :Production changes of competitor(s)
+STR_NEWS_INDUSTRY_NOBODY                                        :Production changes of other industries
+STR_NEWS_ADVICE                                                 :Vehicle news of the company
+STR_NEWS_NEW_VEHICLES                                           :New vehicle available
+STR_NEWS_ACCEPTANCE                                             :Cargo acceptance changes
+STR_NEWS_SUBSIDIES                                              :Subsidies
+STR_NEWS_GENERAL                                                :General news (from towns)
+
 STR_MESSAGE_NEWS_FORMAT                                         :{STRING}  -  {STRING5}
 
 STR_NEWS_MESSAGE_CAPTION                                        :{WHITE}Message
